java/ql/automodel/publish.sh

```diff
@@ -97,9 +103,13 @@ pushd "$WORKSPACE_ROOT"
 echo "Preparing the release"
 "${CODEQL_DIST}/codeql" pack release --groups $GRPS -v
 
-echo "Publishing the release"
-# Add --dry-run to test publishing
-"${CODEQL_DIST}/codeql" pack publish --groups $GRPS -v
+if [ "$1" = "--dry-run" ] || [ "$2" = "--dry-run" ]; then
```

Comment:
see previous comment - this could be simplified to `if [ $DRY_RUN = "1" ]; then`